src/core/handlers/GraphQLHandler.test.ts

```diff
@@ -26,19 +26,19 @@ const resolver: ResponseResolver<GraphQLResolverExtras<{ userId: string }>> = ({
 
 function createGetGraphQLRequest(
   body: GraphQLRequestBody<any>,
-  hostname = 'https://example.com',
+  graphqlEndpoint = 'https://example.com',
```

Comment:
kind of nitty, but I change the argument name here

hostname implies `example.com` to me, and I wanted it to be clear this is actual an endpoint.  this could be `graphqlUrl` or `graphqleServerHref` or something, but I think this naming is probably clear enough?

Reply:
Bad naming on my part in the past, great correction.